src/components/interaction-designer/BuilderCanvas.vue

```diff
@@ -23,10 +26,69 @@
 
     computed: {
       ...mapState('flow', ['flows']),
+
+      canvasHeight() {
+        const defaultHeight = window.screen.availHeight
+        if (!this.flows[0].blocks && this.flows[0].blocks.length) {
+          return defaultHeight
+        }
+
+        const blockAtTheLowestPosition = lodash.maxBy(this.flows[0].blocks, 'platform_metadata.io_viamo.uiData.yPosition')
+
+        if (!blockAtTheLowestPosition) {
+          return defaultHeight
+        }
+
+        const blockElement = document.getElementById(`block/${blockAtTheLowestPosition.uuid}`)
+
+        if (!blockElement) {
+          return defaultHeight
+        }
+
+        const blockHeight = blockElement.clientHeight
+        return lodash.get(blockAtTheLowestPosition, 'platform_metadata.io_viamo.uiData.yPosition') + blockHeight + 100
```

Comment:
Can we `import {get} from 'lodash'` here?

Reply:
Can we assign this `100` to a variable so that we have some indication to the fact that it's a buffer beyond our furthest outlying block?

Reply:
Sure, added it as `deltaY`